inspirehep/modules/records/mappings/records/experiments.json

```diff
@@ -65,8 +65,15 @@
                     "index": "not_analyzed"
                 },
                 "affiliation": {
-                    "type": "string",
-                    "index": "not_analyzed"
+                    "type": "object",
+                    "properties": {
+                        "name": {
+                            "type": "string"
+                        },
+                        "curated_relation": {
+                            "type": "boolean"
+                        }
```

Comment:
ES mappings will be cleaned soon by @kaplun, including mappings for all occurances of `json_reference` type.